### DISCUSSION TITLE
Coordinator: switch from EFR32MG21 to CC2652P without re-pairing?

### DISCUSSION THREAD

**Discussion by paolofaz** [Koenkk/zigbee2mqtt#21612] [General]
Hi to all.
I'm using as a cordinator the sonoff E dongle with EFR32MG21 chip.
It Is possibile to switch to CC2652 dongle without re-pairing all device?
Are the two chips more or less equivalent in terms of zigbee performance or is one better?
Thanks
Have good day
Paolo

**Comment by francisp2** [comment]
See here: 
https://github.com/Koenkk/zigbee2mqtt/discussions/16478#discussioncomment-8178265
